src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2070,6 +2070,17 @@ bool AdaptDispatch::SetCursorColor(const COLORREF cursorColor)
     return _pConApi->SetCursorColor(cursorColor);
 }
 
+// Routine Description:
+// - OSC Copy to Clipboard
+// Arguments:
+// - content - The content to copy to clipboard. Must be null terminated.
+// Return Value:
+// - True if handled successfully. False otherwise.
+bool AdaptDispatch::CopyToClipboard(const std::wstring_view content)
+{
+    return _pConApi->CopyToClipboard(content);
```

Comment:
You could probably avoid a lot of plumbing here by just returning false here. When an `AdaptDispatch` method returns false, and the console is working in conpty mode, we'll automatically try to pass the string through to the connected terminal application. That way, you wouldn't need to manually plumb the sequence all the way through `VtIo` into the `VtEngine`, etc 😄

Reply:
I missed this point. That's much simpler.